test/cmdlineTests.sh

```diff
@@ -302,25 +320,28 @@ test_solc_behaviour "${0}" "ctx:=/some/remapping/target" "" "" 1 "" "Invalid rem
 printTask "Running general commandline tests..."
 (
     cd "$REPO_ROOT"/test/cmdlineTests/
-    for tdir in ${*:-*/}
+    if [[ ${#tdirs[@]} == 0 ]]
+    then
+        tdirs=(*)
+    fi
```

Comment:
This can be shortened a bit.
```suggestion
    [[ ${#tdirs[@]} == 0 ]] || tdirs=(*)
```

Reply:
done